build/package/powershell/windows_post_package_test.ps1

```diff
@@ -0,0 +1,21 @@
+# Install current version of MongoDB Atlas CLI
+$NewestVersion = git tag --list ${Env:TOOL_NAME}/v* --sort=taggerdate | tail -1 | cut -d "v" -f 2
+cd ../../../dist/
+$PackageName = "mongodb-atlas-cli_${NewestVersion}_windows_x86_64.msi"
+$Source = "https://fastdl.mongodb.org/mongocli/${PackageName}"
```

Comment:
we should download the msi of the current task as oppose to the one published

Reply:
Updated!